feugy/tabulous
# feat(web): displays ane enforces game max seats

### What's in there?

Before allowing players to share a game with a simple URL, Tabulous needs a mechanism to avoid too many connected players.
This PR introduces `minSeats`  and `maxSeats`  in game descriptors, which enable `availableSeats` computation on the backend.
It also allows to display a number of players in the game catalog.
In the game page, the menu only shows the invite option and dialogue if there are available seats.

Are included here:

- feat(web): query and displays copyright and player ranges
- feat(web): do not dispay invite option when there are no available seats
- feat(games): adds seats to public games
- feat(games): renames "coinche" in "32-cards" generic game
- refactor(server): computes available seats on server side
- refactor(web): completely removes closed dialogues instead of hiding them
- test(web): uses aria roles instead of css selectors
- test(web): enforces using a single Playwright worker for global initialization
- chore(web): fixes hot module reloading with vite

### How to test?

1. browse to https://localhost:3000
   > you should see min and max seats on public games
1. open a draughts game
1. open game menu, and invite another player
1. open the game menu once more
   > the invite option should be hidden
1. reload the game page and option the game menu
   > the invite option should still be hidden
